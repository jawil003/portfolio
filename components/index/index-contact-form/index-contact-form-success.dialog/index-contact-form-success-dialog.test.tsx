import { render } from "@testing-library/react";
import React from "react";
import {
  IndexContactFormSuccessDialog,
  IndexContactFormSuccessDialogProps,
} from ".";

const variants: IndexContactFormSuccessDialogProps[] = [
  { mode: "success" },
  { mode: "error" },
];

describe("IndexContactFormSuccessDialogSuccess", () => {
  test.each(variants)(
    "Create IndexContactFormSuccessDialogSuccess with %p",
    (variant) => {
      const testDialog = render(
        <IndexContactFormSuccessDialog
          {...variant}
        />,
      );

      expect(
        testDialog,
      ).toMatchSnapshot();
    },
  );
});
