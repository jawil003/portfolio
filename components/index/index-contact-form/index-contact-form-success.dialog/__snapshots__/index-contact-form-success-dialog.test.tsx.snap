// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexContactFormSuccessDialogSuccess Create IndexContactFormSuccessDialogSuccess with {"mode": "error"} 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .emotion-0 {
  padding: 50px 30px;
  background-color: #f3f3f3;
  border-radius: 20px;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-1.emotion-1 {
  fill: red;
  font-size: 175px;
}

.emotion-2 {
  text-align: start;
  font-style: normal;
  margin: unset;
  -webkit-text-decoration: none;
  text-decoration: none;
  letter-spacing: normal;
  word-spacing: normal;
  color: #000;
  word-wrap: unset;
  padding: unset;
  word-break: normal;
}

.emotion-2 {
  font-weight: 400;
  font-size: 1.5em;
}

<body>
    <div>
      <div
        class="emotion-0"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root emotion-1"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"
          />
        </svg>
        <div
          style="height: 20px;"
        />
        <span
          class="emotion-2 emotion-3"
          color="#000"
          id="heading5"
        >
          Etwas ist schiefgelaufen
        </span>
      </div>
    </div>
  </body>,
  "container": .emotion-0 {
  padding: 50px 30px;
  background-color: #f3f3f3;
  border-radius: 20px;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-1.emotion-1 {
  fill: red;
  font-size: 175px;
}

.emotion-2 {
  text-align: start;
  font-style: normal;
  margin: unset;
  -webkit-text-decoration: none;
  text-decoration: none;
  letter-spacing: normal;
  word-spacing: normal;
  color: #000;
  word-wrap: unset;
  padding: unset;
  word-break: normal;
}

.emotion-2 {
  font-weight: 400;
  font-size: 1.5em;
}

<div>
    <div
      class="emotion-0"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root emotion-1"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"
        />
      </svg>
      <div
        style="height: 20px;"
      />
      <span
        class="emotion-2 emotion-3"
        color="#000"
        id="heading5"
      >
        Etwas ist schiefgelaufen
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`IndexContactFormSuccessDialogSuccess Create IndexContactFormSuccessDialogSuccess with {"mode": "success"} 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .emotion-0 {
  padding: 50px 30px;
  background-color: #f3f3f3;
  border-radius: 20px;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-1.emotion-1 {
  fill: #28b463;
  font-size: 175px;
}

.emotion-2 {
  text-align: start;
  font-style: normal;
  margin: unset;
  -webkit-text-decoration: none;
  text-decoration: none;
  letter-spacing: normal;
  word-spacing: normal;
  color: #000;
  word-wrap: unset;
  padding: unset;
  word-break: normal;
}

.emotion-2 {
  font-weight: 400;
  font-size: 1.5em;
}

<body>
    <div>
      <div
        class="emotion-0"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root emotion-1"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
          />
        </svg>
        <div
          style="height: 20px;"
        />
        <span
          class="emotion-2 emotion-3"
          color="#000"
          id="heading5"
        >
          Erfolgreich abgesendet
        </span>
      </div>
    </div>
  </body>,
  "container": .emotion-0 {
  padding: 50px 30px;
  background-color: #f3f3f3;
  border-radius: 20px;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-1.emotion-1 {
  fill: #28b463;
  font-size: 175px;
}

.emotion-2 {
  text-align: start;
  font-style: normal;
  margin: unset;
  -webkit-text-decoration: none;
  text-decoration: none;
  letter-spacing: normal;
  word-spacing: normal;
  color: #000;
  word-wrap: unset;
  padding: unset;
  word-break: normal;
}

.emotion-2 {
  font-weight: 400;
  font-size: 1.5em;
}

<div>
    <div
      class="emotion-0"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root emotion-1"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
        />
      </svg>
      <div
        style="height: 20px;"
      />
      <span
        class="emotion-2 emotion-3"
        color="#000"
        id="heading5"
      >
        Erfolgreich abgesendet
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
